addons/l10n_nl/models/res_partner.py

```diff
@@ -0,0 +1,9 @@
+# coding: utf-8
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    l10n_nl_kvk = fields.Char(string='KVK-nummer')
+    l10n_nl_oin = fields.Char(string='Organisatie Indentificatie Nummer')
```

Comment:
Only when you send it digitally?

Reply:
Can you elaborate ? :D Otherwise, see my other comment https://github.com/odoo/odoo/pull/68514#discussion_r604769021

Reply:
Checking online, this field is only used for sending digitally.  (You need this number if you want to send it electronically to the government and I don't think we support this yet.)